src/applications/disability-benefits/526EZ/reducer.js

```diff
@@ -1,6 +1,48 @@
+
+import _ from 'lodash/fp';
+
 import formConfig from './config/form';
 import { createSaveInProgressFormReducer } from '../../common/schemaform/save-in-progress/reducers';
 
+import {
+  PRESTART_STATUS_SET,
+  PRESTART_DATA_SET,
+  PRESTART_RESET,
+  PRESTART_DISPLAY_RESET,
+  PRESTART_STATUSES
+} from './actions';
+
+const initialState = {
+  status: PRESTART_STATUSES.notAttempted,
+  data: {
+    currentExpirationDate: null,
+    previousExpirationDate: null
+  },
+  display: false
```

Comment:
@cehsu explained it to me in another comment: The idea behind this is to use it to show the success banner or not since we only want to show it on the first page that the veteran sees when they load the form. Once they move to the next page, the banner doesn't need to stay up.